### PR TITLE
Add nessie_server role and playbooks

### LINKED ISSUE #24: Add Nessie
Integrate [Nessie](https://projectnessie.org/) - Transactional Catalog for Data Lakes.

Nessie can be used by [Spark via Iceberg](https://projectnessie.org/tools/iceberg/spark/) or via [CLI](https://projectnessie.org/tools/cli/).

### PR DESCRIPTION
**Pull request by sergkudinov**
#### Which issue(s) this PR fixes:

Fixes #24

#### Additional comments:

I am not sure about installing Nessie server on `edge-01` configured in https://github.com/TOSIT-IO/tdp-collection-extras/blob/master/roles/ansible-tdp-common-actions/templates/ansible-hosts.j2

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Comment by sergkudinov** [comment]
> * The ansible group name `nessie_server` would be better shortened to just `nessie`. The server is the hardware and if anything it would be _nessie_service_ (which isn't aligned with the other ansible group names in the ansible hosts file)

I would continue with `nessie_server`. In some docs, the authors of the project call it as "Nessie server" (https://projectnessie.org/try/configuration/ and https://projectnessie.org/try/server-upgrade/). Moreover, there is a Nessie CLI later take the name like `nessie_cli`. So, I suggest to keep `nessie_server`.

File: tdp_lib_dag/nessie.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
[the remark this answers is not in the document]
Comment (sergkudinov) [reply tdp_lib_dag/nessie.yml:1]: To test this, how can I install `tdp` and try it? Is it this doc https://github.com/TOSIT-IO/tdp-lib/blob/master/docs/developer/cli/developer_quick_start.rst ?

[the remark this answers is not in the document]
Comment (rpignolet) [reply tdp_lib_dag/nessie.yml:1]: Yes and you need to set `export TDP_COLLECTION_PATH="~/tdp-collection:~/tdp-collection-extras"`.
We do not have command to validate the DAG, every `tdp` command that use the DAG perform the DAG validation.

**Comment by leopaul36** [comment]
Closing as stale

**Pull request closed without merging**